test/ui/load40-cache-on.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Loads users/ page 40 times (with caching on) capturing a screenshot and load time for each.
+ */
+
+ const puppeteer = require("puppeteer");
+ const testCount = 40;
+ const url = "https://nsc-thursday-react-app.azureedge.net/users";
+
+ (async () => {
+  const browser = await puppeteer.launch();
+
+  const page = await browser.newPage();
+  page.setCacheEnabled([true]);
+
+  for (let i = 0; i < testCount; ++i){
+    await page.goto(url);
```

Comment:
Some of your screenshots haven't loaded: 
![image](https://user-images.githubusercontent.com/57076614/112560975-82d98580-8d91-11eb-8c61-0adf015e6c32.png)
I found that changing the `page.goto(url)` to `page.goto(url, waitUntil="networkidle2")` fixed this - another option is "networkidle0" but I didn't find that to be necessary to get the loaded data.